app/src/main/java/com/himanshuvirmani/androidreactivearch/base/dependencyinjection/components/ApplicationComponent.java

```diff
@@ -41,5 +47,12 @@
 
   CacheManager cacheManager();
 
-  ApiManager apiManager();
+  //ApiManager apiManager();
+
+  // Added model specific methods
+  PostModel postModel();
```

Comment:
Can we avoid writing models as it will lead to an infinite addition to application component for each model